lib/simplecov/result_merger.rb

```diff
@@ -54,20 +54,24 @@ def results
         results
       end
 
+      # Merge two or more SimpleCov::Results into a new one with merged
+      # coverage data and the command_name for the result consisting of a join
+      # on all source result's names
+      def merge_results(*results)
```

Comment:
@PragTob @aroben how about

```ruby
def merge_results(*results = results())
```

this is something that I know has been done in Rails at Matz's recommendation when the default value for a param is a method by the same name